library/src/main/java/com/klinker/android/link_builder/TouchableSpan.java

```diff
@@ -36,20 +37,26 @@ public TouchableSpan(Context context, Link link) {
         this.link = link;
 
         if (link.getTextColor() == 0) {
-            this.textColor = getDefaultColor(context);
+            this.textColor = getDefaultColor(context, R.styleable.LinkBuilder_defaultLinkColor);
         } else {
             this.textColor = link.getTextColor();
         }
+
+        if (link.getTextColorHighlight() == 0) {
+            this.textColorHighlight = getDefaultColor(context, R.styleable.LinkBuilder_defaultLinkHighlightColor);
+        } else {
+            this.textColorHighlight = link.getTextColorHighlight();
+        }
     }
 
     /**
      * Finds the default color for links based on the current theme.
      * @param context activity
      * @return color as an integer
      */
-    private int getDefaultColor(Context context) {
+    private int getDefaultColor(Context context, int index) {
```

Comment:
Please add to the Javadocs, and instead of index, lets call this `res` since it is taking a resource, doesn't have much to do with an index, in my opinion